src/sonic-yang-models/yang-models/sonic-neigh.yang

```diff
@@ -31,14 +27,14 @@ module sonic-neigh {
 				key "vlan neighbor";
 
 				leaf vlan {
-					description "Neighbor interface and IP address ex. Vlan1000|100.1.1.3";
-					type leafref {
-						path "/svlan:sonic-vlan/svlan:VLAN/svlan:VLAN_LIST/svlan:name";
+					description "Neighbor Vlan interface ex. Vlan1000";
```

Comment:
https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-vxlan.yang#L87-L90
Would please add comment like above example?
When we fix back-links issue, we need to remove this workaround.